tests/unit/customElements.ts

```diff
@@ -321,28 +324,29 @@ registerSuite({
 		}
 	},
 
-	'appender': function () {
-		let sandbox: any;
-
-		return {
-			'beforeEach'() {
-				sandbox = sinon.sandbox.create();
-			},
+	'appender': {
+		beforeEach() {
+			sandbox = sinon.sandbox.create();
+		},
 
-			afterEach() {
-				sandbox.restore();
-			},
+		afterEach() {
+			sandbox.restore();
+		},
 
+		tests: {
 			'appender is returned as a function'(this: any) {
 				let rendered = false;
 
 				const appendStub = sandbox.stub();
 
-				sandbox.stub(projector, 'ProjectorMixin', function () {
-					return {
-						append: appendStub
-					};
-				});
+				const OrigProjectorMixin = projector.ProjectorMixin;
```

Comment:
Was this causing an issue as it was?

Reply:
The stubbed ProjectorMixin was reported as not being a constructor.
```
  TypeError: projector is not a constructor
      at Object.initializeElement  <src/customElements.ts:244:24>
      at Test.appender is returned as a function [as test]  <tests/unit/customElements.ts:358:21>
      at <src/lib/Test.ts:257:48>
      at <node_modules/intern/node_modules/@dojo/core/async/Task.ts:319:12>
      at handler  <node_modules/intern/node_modules/@dojo/core/async/ExtensiblePromise.ts:216:14>
      at <anonymous>
```